src/cockpit/channels/dbus.py

```diff
@@ -289,7 +289,7 @@ def sync_handler(message):
 
     async def do_call(self, message):
         path, iface, method, args = message['call']
-        timeout = message.get('timeout')
+        timeout = message.get('timeout', 2**63)
```

Comment:
This feels ugly. `timeout` is an `uint64_t`, so this *should* be safe on 32 bit systems.

I see that systemd_ctypes defaults to 0:

    super().call_async(pending, message, pending.callback, pending.userdata, timeout or 0)

[sd_bus_call_async()](https://man.archlinux.org/man/sd_bus_call_async.3.en) indeed does not have a mode for "indefinite timeout" -- that's sad, I would have liked "0" for "indefinite" and "-1" for "default".

So this is good enough for me. `2**64 -1` aka `MAX_UINT64` (like in pkg/systemd/services/services.jsx) would be a bit more standard, but as it's not a magic value, it doesn't make much difference either way.

Reply:
Yeah, `2**64-1` should work, but something made me want to play it safe... Let's just use `2**64-1`.